### ISSUE TITLE
chaining rule doesn’t work when chain is imported from lodash/chain

### ISSUE THREAD

**Issue by jaydenseric** [wix/eslint-plugin-lodash#246]
The [`lodash/chaining`](https://github.com/wix/eslint-plugin-lodash/blob/master/docs/rules/chaining.md) rule doesn’t work when using a deep import, which is a best practice for bundle-size:

```js
import chain from 'lodash/chain'
```

It only works when `chain` is imported as a named import from `lodash`:

```js
import { chain } from 'lodash'
```

Or when `_.chain` is used from the main default import:

```js
import _ from 'lodash'

_.chain(
  // …
) // …
```

**Comment by blimmer** [comment]
As a workaround you can use `no-restricted-imports`:

```js
"no-restricted-imports": [
  "error",
  {
    "paths": [
      {
        // lodash/chaining should catch this, but there's a bug: https://github.com/wix/eslint-plugin-lodash/issues/246
        "name": "lodash/chain",
        "message": "Do not use lodash/chain, as it breaks tree-shaking. Please nest your calls to lodash."
      }
    ]
  }
]
```

**Comment by vasilismantz** [comment]
Any updates on this?
